node/node.go

```diff
@@ -310,6 +344,12 @@ func (n *Node) run(ctx context.Context) (err error) {
 			n.role = certUpdate.Role
 			n.roleCond.Broadcast()
 			n.Unlock()
+			tlsIssuerInfo := securityConfig.IssuerInfo()
+			n.notifyTLSChange <- &api.NodeTLSInfo{
+				CertIssuerPublicKey: tlsIssuerInfo.PublicKey,
+				CertIssuerSubject:   tlsIssuerInfo.Subject,
+				TrustRootHash:       securityConfig.RootCA().Digest.String(),
+			}
```

Comment:
There's a potential deadlock here if the agent exits before this goroutine does. Put this in a select along with `<-agentDone`.